src/metrics/management/commands/process_crossref_events.py

```diff
@@ -11,62 +11,105 @@
 from identifiers import models as ident_models
 from journal import models as journal_models
 
+CROSSREF_TEMP_DIR = os.path.join(settings.BASE_DIR, 'files', 'temp', 'crossref')
 
-def process_events():
 
-    for journal in journal_models.Journal.objects.all():
-        if journal.use_crossref:
-            crossref_prefix = journal.get_setting('Identifiers', 'crossref_prefix')
+def process_events(journal_prefixes):
```

Comment:
This function has too many nested loops and branches. a separate function to parse a single line and/or an event would help and would make testing easier